doc/source/faq/index.rst

```diff
@@ -438,11 +438,10 @@ which would produce debug (as well as info, warning, and error) messages, or at
 
 .. code-block:: python
 
-   from yt.funcs import mylog
-   mylog.setLevel(40) # This sets the log level to "ERROR"
+   yt.set_log_level("error")
```

Comment:
Since the text below maps the numerical values with text-based log levels, you I would suggest that you add another line that says something to the effect of 

> this is the same as doing:
> 
> yt.set_log_level(40)

and then having the block of text below, which shows the mapping.

Reply:
sure, sounds good

Reply:
done !

Reply:
🎉 😎 🥇